lib/mail_catcher/mail.rb

```diff
@@ -157,4 +157,13 @@ def delete_message!(message_id)
     @delete_messages_query.execute(message_id) and
     @delete_message_parts_query.execute(message_id)
   end
+
+  def delete_older_messages!(max_mail_to_keep = MailCatcher::options[:max_mail_to_keep])
+    return if max_mail_to_keep <= 0
+    @delete_older_messages_query ||= db.prepare "DELETE FROM message WHERE id NOT IN (SELECT id FROM message ORDER BY created_at DESC LIMIT ?)"
+    @delete_older_message_parts_query ||= db.prepare "DELETE FROM message_part WHERE id NOT IN (SELECT id FROM message_part ORDER BY created_at DESC LIMIT ?)"
```

Comment:
Are the right records getting deleted here?
Should this delete the same rows as the previous line?

```sql
DELETE from message_part
WHERE message_id NOT IN (SELECT id from message ORDER BY created_at DESC LIMIT ?)
```

(or `ORDER BY id` if the previous `ORDER BY` change was made)

Reply:
You are right, I change this :).